index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>ArtStation | SignUp</title>
```

Comment:
The name of the current page should come before the brand name, because when the tabs get smaller, the user can only see the first word of the title, so they won't be able to recognize which page is open right now

Reply:
made chnages, thanks!